src/components/side-by-side/side-by-side.tsx

```diff
@@ -34,19 +36,25 @@ export class SideBySide extends Component<SideBySideComponentProps> {
     }
     return (
       <div className={styles.innerButtons}>
-        <div className={styles.iconContainer} onMouseUp={onPIPSwitch}>
+        <Button className={styles.iconContainer} onClick={onPIPSwitch}>
           <Icon id="dualscreen-side-by-side-pip" height={24} width={24} path={icons.SWITCH_TO_SIDE_BY_SIDE_ICON_PATH} />
-        </div>
+        </Button>
       </div>
     );
   }
 
-  render({guiClientRect}: SideBySideComponentProps) {
+  render({guiClientRect, animated}: SideBySideComponentProps) {
     const playerContainerStyles = {
       height: (guiClientRect?.width! / 2 / 16) * 9
     };
+    const classNames = [styles.secondaryPlayer];
+    if (animated) {
+      classNames.push(styles.animatedScale);
+    } else {
+      classNames.push(styles.animatedFade);
+    }
```

Comment:
Boolean name is not representing correctly as both if and else do animations of some kind so they are both "animated"